clusterloader2/testing/load/experimental-config.yaml

```diff
@@ -403,7 +403,7 @@ steps:
 # BEGIN scheduler throughput
 - name: Creating scheduler throughput measurements
   measurements:
-  - Identifier: HighThroughputStartupLatency
+  - Identifier: PodStartupHighThroughputLatency
```

Comment:
HighThroughputPodStartupLatency? :)

HighThroughput refers to pods not to latency

Reply:
I've send inconsistent PR. It's already HighThroughputPodStartupLatency.